main.js

```diff
@@ -0,0 +1,31 @@
+function Add(numbers = "") {
+    if (numbers.trim() == "") return 0;
+    let self = numbers.trim().split(',')
```

Comment:
we should use descriptive variables. 

self has no meaning. 

`const splittedNumbers `